codebuild/run_canary_china.yaml

```diff
@@ -2,6 +2,8 @@ version: 0.2
 phases:
   build:
     commands:
+      # make all shell scripts executable. This is required when running code copyied from S3
+      - find ./ -type f -name "*.sh" -exec chmod +x {} \;
```

Comment:
nit: `\;` at the end is not necessary.

Reply:
it is

Reply:
`\` denotes the command continues to the next line? and then `;` denotes the end of the command? Is this a weird function of the `-exec` flag?

Reply:
ya so the syntax is like this 
`find <something> -exec <command> ;`

but if you just type `;` at it will think that it's part of the <command>
("chmod +x {}" ; does not work either)
so you need to escape it with `\;`

more info 
https://stackoverflow.com/questions/25142822/error-find-exec-no-terminating-or
https://ss64.com/bash/find.html